src/entities/ItemCardDetails/ui/ItemCardDetails.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react';
+
+interface Props {
+  status?: string;
+  synopsis?: string;
+  onCloseDetails?: () => void;
+  id?: number;
+}
```

Comment:
Props in interfaces are also usually sorted. Here's one of examples of sorting rules:
```
interface {
  /* required (no ?) data fields */
  /* required (no ?)  booleans e.g. isOpen, isDisabled */
  /* required (no ?) functions */
   /* optional (?) data fields */
  /* optional (?)  booleans e.g. isOpen, isDisabled */
  /* optional (?) functions */
}
```
But when u're destructing props in component, u can ignore prop optionality and use just:
```
 component = ({
  /*  data fields */
  /*  booleans e.g. isOpen, isDisabled */
  /*  functions */
}) => 
```